### ISSUE TITLE
Unassigned points in View and Filtered Merge View don't always get assigned a point when merged.

### ISSUE THREAD

**Issue by IsaacNPsmash** [MathNetDev/MathNetClient#28]
Tested 9/14/18 @ 9:25
Sent view: Three points creating a circle conic.
Groups: Three groups with only one student in each (3 students total) controlling 1 point, each different from the others, i.e. student1 in group1 controls point A, student2 in group2 controls point B, student3 in group3 controls point C. Each student moves their point creating 3 circles intersecting circles.
Merged view (both View and Filtered): Group1 has all three points visible, assigned "isaac1g1," "unassigned1g1," and "unassigned1g1," but for Group2 and Group3, only "isaac2g2" and "isaac3g3" points are visible and assigned a name.

**Comment by ghost** [comment]
@IsaacNPsmash Currently this has been fixed only on the merged view tab and not the filtered merge view tab. If needed can fix it there as well.

**Comment by IsaacNPsmash** [comment]
Tested 10/1/2018 @ 3:20 Test branch
I am still getting where some of the unassigned points are missing, specifically the point that group 1 moved.
Detail 1: Similar to above, three groups with one student in each; sent view is three point creating a circle conic. Each of the three students moves one point leaving 2 unassigned points in each group. In the merged view, the unassigned points for group1 come through as "unassignedg1" but the point that student1 moved in group1 but the others have as unassigned is not there. 
I also did this with a 5 pointed conic to the same results.

**Comment by ghost** [comment]
@IsaacNPsmash I'm not sure but I don't see that issue. Maybe I'm misunderstanding something or not doing it correctly. How about we have a video call sometime where you can show me the issue? Thanks.

**Comment by IsaacNPsmash** [comment]
Tested 10/11/18 @ 8:37 am Test branch (View tab)
Seems to be working. Not sure if what I was testing before was not the right screen or hadn't updated yet. I did two tests to check to make sure (in regular view) all unassigned points were shown. 
Test 1: Three points, a line between two of them, and a parabola defined by the line and extra point.
Three students in three different groups.
Each student moved a different point. Thus within each group, two unassigned points, and between each group, intersection of one unassigned point.
In merged view, all points are shown, both assigned and unassigned.
Test2: Three points and a circle defined by them.
Each student moved a different point. Thus within each group, two unassigned points, and between each group, intersection of one unassigned point.
In merged view, all points are shown, both assigned and unassigned.
